src/otx/core/data/entity/visual_prompting.py

```diff
@@ -153,10 +153,10 @@ class ZeroShotVisualPromptingBatchDataEntity(OTXBatchDataEntity[ZeroShotVisualPr
         prompts (list[list[tv_tensors.TVTensor]]): List of prompts.
     """
 
-    masks: list[tv_tensors.Mask]
-    labels: list[LongTensor]
-    polygons: list[list[Polygon]]
-    prompts: list[list[tv_tensors.TVTensor]]
+    masks: list[tv_tensors.Mask | None]
+    labels: list[LongTensor | None]
+    polygons: list[list[Polygon] | None]
+    prompts: list[list[tv_tensors.TVTensor] | None]
```

Comment:
Why does everything be changed to be able to `None`?

Reply:
For standalone `infer`, target images will be coming as inputs only.
This implementation is to consider the situation that other inputs are not coming.

Reply:
If so, why not giving empty lists?

Reply:
🫨I didn't think about that, done at https://github.com/openvinotoolkit/training_extensions/pull/3090/commits/6d2a514b4560da8aa8e3349389e3188f1bba518f.